packages/ERTP/test/unitTests/test-mintObj.js

```diff
@@ -104,3 +104,60 @@ test('mint.mintPayment inviteMathHelpers', t => {
     t.end();
   }
 });
+
+
+// Tests related to non-fungible tokens
+// This test models ballet tickets
+test('non-fungible tokens example', t => {
+  try {
+    const { mint: balletTicketMint, issuer: balletTicketIssuer, amountMath } = produceIssuer('Agoric Ballet Opera tickets', 'handle');
+
+    const startDateString = (new Date(2020, 1, 17, 20, 30)).toISOString();
+
+    const ticketDescriptionObjects = Array(5).fill().map((_, i) => ({
+      seat: i+1,
+      show: 'The Sofa',
+      start: startDateString,
+    }))
+
+    const balletTicketPayments = ticketDescriptionObjects.map(ticketDescription => {
+      const handle = {};
+      const amount = amountMath.make(harden([handle]))
+      return balletTicketMint.mintPayment(amount, JSON.stringify(ticketDescription))
```

Comment:
This is really good for us to discover that the memo might be used in this way. I think we would want to make the extents useful enough that information about what the digital assets are, is all in the extent. The memo would be to mostly distinguish between payments for fungible amounts (this $1000 is for next month's rent). Can you think of anything that would help us make this distinction and discourage uses we didn't intend?

Reply:
I think this is outdated after the last discussion